tests/x509/test_x509.py

```diff
@@ -5777,3 +5777,24 @@ def test_no_attributes(self, backend):
             backend,
         )
         assert len(request.attributes) == 0
+
+
+def test_load_pem_x509_certificates():
+    with pytest.raises(ValueError):
+        x509.load_pem_x509_certificates(b"")
+
+    certs = load_vectors_from_file(
+        filename=os.path.join("x509", "cryptography.io.chain.pem"),
+        loader=lambda pemfile: x509.load_pem_x509_certificates(pemfile.read()),
+        mode="rb",
+    )
+    assert len(certs) == 2
+
+    certs = load_vectors_from_file(
+        filename=os.path.join(
+            "x509", "cryptography.io.chain_with_garbage.pem"
+        ),
+        loader=lambda pemfile: x509.load_pem_x509_certificates(pemfile.read()),
+        mode="rb",
+    )
+    assert len(certs) == 2
```

Comment:
Please add some basic assertion about the certs.

Reply:
Just clarifying: do you want me to assert that these are actually certificates, or do some basic attribute checks on them to make sure we're getting something sensible back in each?

Reply:
yeah, check attrs to make sure we got hte right certs -- serial number or something simple.

Reply:
Sounds good, will add in a moment.

Reply:
7bde98a25